man/systemd.exec.xml

```diff
@@ -1327,6 +1327,12 @@ CapabilityBoundingSet=~CAP_B CAP_C</programlisting>
         configuration or lifetime guarantees, please consider using
         <citerefentry><refentrytitle>tmpfiles.d</refentrytitle><manvolnum>5</manvolnum></citerefentry>.</para>
 
+        <para><varname>RuntimeDirectory=</varname> and <varname>StateDirectory=</varname> optionally support
+        a second parameter, separated by <literal>:</literal>. The second parameter will be interpreted as a
+        destination path that will be created as a symlink to the directory. The symlinks will be created
+        after any <varname>BindPaths=</varname> or <varname>TemporaryFileSystem=</varname> options have been
+        set up, to make ephemeral symlinking possible. The same source can have multiple symlinks, </para></listitem>
```

Comment:
maybe extend the last sentence by saying "by using the setting multiple times with the same first parameter, but diferent second parameters"